.github/workflows/test_notebook.yml

```diff
@@ -4,13 +4,19 @@ on:
   pull_request:
     branches: [main]
     paths:
-      - '**.py'
-      - '**.ipynb'
+     - "pyproject.toml"
+     - "tests/**.py"
+     - "pymc_marketing/**"
+     - "docs/source/notebooks/**.ipynb"
+     - "!docs/source/notebooks/*/dev/**.ipynb"
   push:
     branches: [main]
     paths:
-      - '**.py'
-      - '**.ipynb'
+     - "pyproject.toml"
+     - "tests/**.py"
+     - "pymc_marketing/**"
+     - "docs/source/notebooks/**.ipynb"
+     - "!docs/source/notebooks/*/dev/**.ipynb"
```

Comment:
I did not know this negation was possible :)

Reply:
In the docs [here](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths)